cfme/utils/blockers.py

```diff
@@ -181,6 +181,33 @@ def bugzilla_bug(self):
             return None
         return self.data
 
+    @property
+    def can_test_on_current_upstream_appliance(self):
```

Comment:
@mshriver and I discussed a name change for this function and also making it free from references to the appliance, with the appliance specific stuff coming in the `blocks` function.

Reply:
On second glance, however, the property `can_test_on_upstream` already provides a function free from references to the appliance. This property is designed for use within the `blocks` function, and therefore, needs the reference to the appliance. 

It is answering the question: "if I have a bug, and that bug is in a "POST" or "MODIFIED" state, and if I have an upstream appliance, is the build date of my current upstream appliance after the fix for the bug was merged?" If so, then it will return `True`, because the bug's fix is included in your current upstream appliance. If you happen to be on an older version of an upstream appliance, then it will return false.

Reply:
Adding a docblock for this function, so that's its intent is more clear